cilium-sysdump/__main__.py

```diff
@@ -77,12 +86,12 @@ def parse_comma_sep_list(arg_string):
                         help='only return logs newer than a '
                              'relative duration like 5s, 2m, or'
                              ' 3h. Defaults to 30m.',
```

Comment:
I think this message needs adjustment now that the default changed to include all logs.

Reply:
Thanks!